### ISSUE TITLE
NTR: Ulcers ileum

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3939]
New term request: Ulcers ileum. Definition: \
Source: \
Parent: \
Disease association:

**Comment by pnrobinson** [comment]
- [ ] THis should be placed here Abnormality of the ileum (https://hpo.jax.org/app/browse/term/HP:0001549)
- [ ] Suggested label: Ileal ulcer
- [ ] Definition: An erosion of the mucous membrane of in a portion of the ileum. [PMID:28670609]

**Comment by pnrobinson** [comment]
adding as HP_0032024
